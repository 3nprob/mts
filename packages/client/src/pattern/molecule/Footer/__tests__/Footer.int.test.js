import { defaultRender } from './Footer.int.render'

describe('<Footer />', () => {
  describe('Snaphot', () => {
    test('must match defaultRender', () => {
      expect(global.renderToJSON(defaultRender)).toMatchSnapshot()
    })
  })
})
