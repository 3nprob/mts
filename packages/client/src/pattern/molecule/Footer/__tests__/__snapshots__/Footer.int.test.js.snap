// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> Snaphot must match defaultRender 1`] = `
<div
  className="footer"
>
  Under Construction
</div>
`;
